Trust and Cooperation (#7)
mietright/code-of-conduct: README.md

Treat others as you wish to be treated.

## Trust and Cooperation
We are on the same team, we do not judge, rather support each other.
Everyone can trust that it is safe to raise concerns and speak up if there is a problem. Everyone can trust that it will be taken seriously and it will not backfire.

> Assume the best intent of your colleagues: social distances due to working remotely can give you the opposite impression.

Reviewer: Can we add a new line between line 12 and 13 to separate the paragraphs?

Same between line 13 and 14?

Reviewer: ```suggestion
Assume the best intentions of your colleagues: social distances due to working remotely can give you the opposite impression.
```

> Everyone can express his opinion without expecting judgment.

Reviewer: ```suggestion
Everyone can express their opinion without expecting judgment.
```

> We do not judge our teammates' opinions, and we assume their best intentions.

Reviewer: Isn't this line repeated while merging the first two paragraphs?

Author: you're right, it can be removed

We help build up our teammates instead of tearing them down.

### Transparency

> We do not hide neither information nor planning to our teammates regarding the development of our projects and features, weather or not they are part of it.

Reviewer: ```suggestion
We do not hide either information or planning to our teammates regarding the development of our projects and features, whether or not they are part of it.
```

We do not hide anything to protect our image or keep control of something.

> We assume our teammates are completely transparent with us.

Reviewer: ```suggestion
We assume our teammates are entirely transparent with us.
```

Treat others as you wish to be treated.

## Trust and Cooperation
We are on the same team, we do not judge, rather support each other.
Everyone can trust that it is safe to raise concerns and speak up if there is a problem. Everyone can trust that it will be taken seriously and it will not backfire.

> Assume the best intentions of your colleagues: social distances due to working remotely can give you the opposite impression.

Reviewer 2: ```suggestion
Assume the best intentions of your colleagues: social distances due to working remotely can give you a different impression than it was meant.
```
Does not always have to be the "opposite" in order to create a miscommunication :)
